src/wheel/bdist_wheel.py

```diff
@@ -35,6 +34,21 @@
 PY_LIMITED_API_PATTERN = r'cp3\d'
 
 
+def python_tag():
+    return 'py{}'.format(sys.version_info[0])
+
+
+# copied from pep425tags.py, is there a better way?
+def get_platform():
+    """Return our platform name 'win32', 'linux_x86_64'"""
+    # XXX remove distutils dependency
+    result = distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    if result == "linux_x86_64" and sys.maxsize == 2147483647:
+        # pip pull request #3497
+        result = "linux_i686"
+    return result
+
+
```

Comment:
It would be nice if packaging exposed this method. I think the XXX is kind of impossible to remove.